bin/index.js

```diff
@@ -26,31 +27,44 @@ if ( path.basename( RELEASE_PATH ) != 'ckeditor4-releases' ) {
 	process.exit( 0 );
 }
 
-function bundlePreset( presetName, version, buildDir ) {
-	var releaseTag = presetName === STANDARD_ALL ? version : `${presetName}/${version}`;
+async function bundlePreset( presetName, version, buildDir ) {
+	const releaseTag = presetName === STANDARD_ALL ? `${ version }-lts` : `${ presetName }/${ version }-lts`;
```

Comment:
It would be good to customize this script so it supports both, LTS and non-LTS versions. For example, we could use the same parameter as for CKBuilder, `--lts`